### PR TITLE
Minimizes the issue template

### LINKED ISSUE #6: Minimize the issue template
Current Issue template is too much bloated

### PR DESCRIPTION
**Pull request by agarwalrounak**
### Checklist:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [x] Have you successfully ran tests with your changes locally?
* [x] Have you added an explanation of what your changes do and why you'd like us to include them?

### Issues Solved

Fixes #6

### CODE REVIEW

**Review by aashutoshrathi — APPROVED** [review]
Looks better

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,15 +1,19 @@
...
 
-### Required Information
-- [ ] Provide a **minimal code snippet** / example that reproduces the bug.
-- [ ] Provide **screenshots** (if required)
-- [ ] Provide github **revision**
+**Steps to reproduce it**
+
+<!--Add steps to reproduce bugs or add information on the place where the feature should be implemented. Add links to a sample deployment or code.-->
+
+**Screenshots of the issue**(if required)
```
Comment (aashutoshrathi) [inline .github/ISSUE_TEMPLATE.md]: Add a space before `(`

**Comment by agarwalrounak** [comment]
Done, please review.

**Review by aashutoshrathi — APPROVED** [review]

**Comment by aashutoshrathi** [comment]
@agarwalrounak Thanks for the PR again :wink:

**Pull request merged**